src/IW/Effects/Cabal.hs

```diff
@@ -0,0 +1,54 @@
+module IW.Effects.Cabal
```

Comment:
@chshersh On it. I've been thinking about documentation and I was wondering if it is better to write the docs for the function you're gonna write first and then implement the function itself?

Reply:
@rashadg1030 Hard to tell :thinking: I think this is individual. Usually, I write documentation only after I've implemented a function. Because during implementation I might realize that I don't even need this function at all! So why spend time on writing docs? :slightly_smiling_face: I've never heard of DDD -- documentation-driven development. But I can imagine that for some people it might be easier to describe the intention in natural language and then implement the thing.